settings.gradle.kts

```diff
@@ -235,6 +235,7 @@ unassigned {
     subproject("soak")
     subproject("smoke-test")
     subproject("performance")
+    subproject("smoke-ide-test")
```

Comment:
@bamboo or we should assign it on configuration-team?

Reply:
Let's assign it to the configuration team with the goal of eventually moving it over to the ide-experience team.

Reply:
@bamboo and the same for the code ownership, I guess?

Reply:
Yes